TS: Compilation fails when importing `rivet-react` into TS app

### The Problem
When importing `rivet-react` 0.4.1 into a TS project, compilation fails with the error,

```
Failed to compile.
Module not found: Error: Can't resolve './DismissibleAlert' in '/Users/jhoerr/Documents/GitHub/azfun-fsharp/app/node_modules/rivet-react/build/dist/components/Alert'
```

On inspection of the `DismissibleAlerts` typings file, `DismissibleAlert.d.ts`, we see:

```
declare const _default: React.StatelessComponent<StatefulAlertProps & React.HTMLAttributes<HTMLDivElement> & import("../src/components/util/Rivet").Props>;
```

This is flagged with a compilation error because the source file `../src/components/util/Rivet` is not present in the package. 

In comparison the `Alert` and `InlineAlert` compile correctly. Their typings read:

```
// Alert
import * as Rivet from '../util/Rivet';
declare const _default: React.StatelessComponent<StatelessAlertProps & React.HTMLAttributes<HTMLDivElement> & Rivet.Props>;

// InlineAlert
import * as Rivet from '../util/Rivet';
declare const _default: React.StatelessComponent<InlineAlertProps & React.HTMLAttributes<HTMLDivElement> & Rivet.Props>;
```

The difference is in how we import `Rivet` into the Components. Here corresponding source TSX for those components:

```
// DismissibleAlert
import { rivetize } from '../util/Rivet';
export default rivetize(DismissibleAlert);

// Alert
import * as Rivet from '../util/Rivet';
export default Rivet.rivetize(Alert);

// InlineAlert
import * as Rivet from '../util/Rivet';
export default Rivet.rivetize(InlineAlert);
```

### Proposed Solution
The fix is to update all affected components to use the `Rivet.rivetize()` method of rivetizing components.

### Scope
The components affected by this issue include:
`DismissibleAlert`, `Badge`, `File`, `Col`, `Container`, `Row`, `Panel`, `Section`, `Table`.

--- a/src/components/Alert/DismissibleAlert.tsx
+++ b/src/components/Alert/DismissibleAlert.tsx
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 import Alert from './Alert'
 import { StatefulAlertProps, statefulPropTypes } from "./common"
 
@@ -23,4 +23,4 @@ class DismissibleAlert extends React.PureComponent<StatefulAlertProps & React.HT
     };
 }
 
-export default rivetize(DismissibleAlert);
+export default Rivet.rivetize(DismissibleAlert);

--- a/src/components/Badge/Badge.tsx
+++ b/src/components/Badge/Badge.tsx
@@ -1,7 +1,7 @@
 import * as classNames from 'classnames';
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 interface BadgeProps {
     /**
@@ -34,4 +34,4 @@ const Badge : React.SFC<BadgeProps & React.HTMLAttributes<HTMLDivElement>> = ({ 
 Badge.displayName = 'Badge';
 Badge.propTypes = propTypes;
 
-export default rivetize(Badge);
+export default Rivet.rivetize(Badge);

--- a/src/components/Dropdown/Dropdown.tsx
+++ b/src/components/Dropdown/Dropdown.tsx
@@ -2,7 +2,7 @@ import * as classNames from 'classnames';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import Button, { ButtonProps } from '../Button/Button';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 import Icon from '../util/RivetIcons';
 import DropdownEvent from './DropdownEvent';
 
@@ -107,4 +107,4 @@ export class Dropdown extends React.PureComponent<DropdownProps & React.HTMLAttr
 
 }
 
-export default rivetize(Dropdown);
+export default Rivet.rivetize(Dropdown);

--- a/src/components/Grid/Col.tsx
+++ b/src/components/Grid/Col.tsx
@@ -1,6 +1,6 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 interface ColProps {
     /**
@@ -143,4 +143,4 @@ const Col : React.SFC<ColProps & React.HTMLAttributes<HTMLDivElement>> =
 }
 Col.displayName = 'Col';
 
-export default rivetize(Col);
+export default Rivet.rivetize(Col);

--- a/src/components/Grid/Container.tsx
+++ b/src/components/Grid/Container.tsx
@@ -1,6 +1,6 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 interface ContainerProps {
     /**
@@ -27,4 +27,4 @@ const Container : React.SFC<ContainerProps & React.HTMLAttributes<HTMLDivElement
 };
 Container.displayName = 'Container';
 
-export default rivetize(Container);
+export default Rivet.rivetize(Container);

--- a/src/components/Grid/Row.tsx
+++ b/src/components/Grid/Row.tsx
@@ -1,6 +1,6 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 const Row : React.SFC<React.HTMLAttributes<HTMLDivElement>> = 
 ({ children, className, ...attrs }) => (
@@ -10,4 +10,4 @@ const Row : React.SFC<React.HTMLAttributes<HTMLDivElement>> =
 );
 Row.displayName = 'Row';
 
-export default rivetize(Row);
+export default Rivet.rivetize(Row);

--- a/src/components/Panel/Panel.tsx
+++ b/src/components/Panel/Panel.tsx
@@ -1,6 +1,6 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 interface PanelProps {
     /**
@@ -21,4 +21,4 @@ const Panel : React.SFC<PanelProps & React.HTMLAttributes<HTMLDivElement>> =
 };
 Panel.displayName = 'Panel';
 
-export default rivetize(Panel);
+export default Rivet.rivetize(Panel);

--- a/src/components/Section/Section.tsx
+++ b/src/components/Section/Section.tsx
@@ -1,9 +1,9 @@
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 const Section : React.SFC<React.HTMLAttributes<HTMLDivElement>> = (props) => (
     <section {...props} />
 );
 Section.displayName = 'Section';
 
-export default rivetize(Section);
+export default Rivet.rivetize(Section);

--- a/src/components/Table/Table.tsx
+++ b/src/components/Table/Table.tsx
@@ -1,6 +1,6 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
-import { rivetize } from '../util/Rivet';
+import * as Rivet from '../util/Rivet';
 
 interface TableProps {
     /** 
@@ -23,4 +23,4 @@ const Table : React.SFC<TableProps & React.HTMLAttributes<HTMLTableElement>> =
 };
 Table.displayName = 'Table';
 
-export default rivetize(Table);
+export default Rivet.rivetize(Table);

